tests/cephadm/test_add_host_to_maintenance_using_flags.py

```diff
@@ -0,0 +1,43 @@
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.utils import get_node_by_id
+from cli.exceptions import OperationFailedError
+
+
+def run(ceph_cluster, **kw):
+    """
+    Create a pool for rbd application that has
+    size and min_size 2 and pool_type replicated.
+    Force a host with OSDs to enter maintenance mode
+    using flags --force and --yes-i-really-mean-it
+    """
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    print(cephadm)
+    orch_obj = RadosOrchestrator(node=cephadm)
+    pool_config = config.get("pool_config")
+    instance = config.get("node")
+    instance_name = get_node_by_id(ceph_cluster, instance).hostname
+
+    # Create a pool
+    out = orch_obj.create_pool(**pool_config)
+    if not out:
+        raise OperationFailedError("Failed to create pool")
+
+    # Put the host into maintainence
+    cmd = f"ceph orch host maintenance enter {instance_name} --force --yes-i-really-mean-it"
+    out = cephadm.shell([cmd])
```

Comment:
Why `cephadm.shell`? Add libraries for same operation

Reply:
Hey @vamahaja  Using this "--yes-i-really-mean-it" flag we are forcing a host into maintenance and it not a recommended way to do it. 
Not sure if it will be used in other test scenarios, please let me know if still this needs to converted to a lib.

Reply:
Done